src/components/ebay-tri-state-checkbox/examples/04-mixed-implementation/template.marko

```diff
@@ -0,0 +1,70 @@
+import ebayCheckbox from "<ebay-checkbox>"
+import ebayMultiCheckbox from "<ebay-tri-state-checkbox>"
+
+class {
+    onCreate() {
+        this.state = {
+            count: 2,
+            checked: [false, true, true, false],
+            cachedChecked: [false, true, true, false],
+            cacheCount: 2
+        };
+    }
+
+    handleChange(index, { checked }) {
+        this.state.checked[index] = checked;
+        this.state.cachedChecked = [...this.state.checked];
+        if (checked) {
+            this.state.count += 1;
+        } else {
+            this.state.count -= 1;
+        }
+        this.state.cacheCount = this.state.count;
+    }
+    handleParentChange({ checked }) {
+        let count;
+        if (checked === "true" || checked === true) {
+            this.state.checked.fill(true);
+            count = this.state.checked.length;
+        } else if (checked === "mixed") {
+            this.state.checked = [...this.state.cachedChecked];
+            count = this.state.cacheCount;
+        } else {
+            this.state.checked.fill(false);
+            count = 0;
+        }
+        this.state.count = count;
+    }
+
+    get checked() {
+        return this.state.count === 0
+            ? "false"
+            : this.state.count === this.state.checked.length
+            ? "true"
+            : "mixed";
+    }
+}
+
+<div class="field">
+    <ebay-tri-state-checkbox
+        skip-mixed=(state.cacheCount === 0 || state.cacheCount === state.checked.length)
```

Comment:
Can we abstract out `(state.cacheCount === 0 || state.cacheCount === state.checked.length)` to a function? e.g. `skip-mixed=shouldSkipMixed()`. It would makes this portion of the example easier to follow.